style/app.scss

```diff
@@ -687,131 +642,121 @@ a:active
 
 #sidebar
 {
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #7a6a96;
-  position         : absolute;
-  padding-left     : 23px;
+  display          : flex;
+  flex-direction   : column;
+  z-index          : 1;
+  position         : inherit;
+  width            : 140px;
+  background-color : #ffffff;
+  border-radius    : 20px;
+  box-shadow       : 2px 2px 2px #e8e9eb;
```

Comment:
I did the same for FCE count in order to meld the two more and make it look more cohesive as one piece rather than 2.
<img width="745" alt="Screen Shot 2021-07-26 at 9 59 55 AM" src="https://user-images.githubusercontent.com/32244859/127001486-951e9933-90eb-4cd0-99bb-c7a89c952c19.png">
<img width="714" alt="Screen Shot 2021-07-26 at 10 00 10 AM" src="https://user-images.githubusercontent.com/32244859/127001527-ada53d22-091b-4ac9-b03b-8405705899dc.png">